src/scripts/achievements/AchievementHandler.ts

```diff
@@ -41,14 +41,19 @@ class AchievementHandler {
         return this.achievementListFiltered().slice(this.navigateIndex() * 10, (this.navigateIndex() * 10) + 10);
     }
 
-    public static filterAchievementList() {
+    public static filterAchievementList(retainPage?: boolean) {
```

Comment:
Can we make this explicitly false, instead of falsey
```suggestion
    public static filterAchievementList(retainPage = false) {
```